checkov.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: checkov
-  version: 3.0.34
+  version: 3.2.257
```

Comment:
The Git commit SHA would also need to be updated, i.e line 29 in the melange. i.e for 3.2.257, this would need to match the SHA associated with that release:
 - https://github.com/bridgecrewio/checkov/releases/tag/3.2.257

Automation would normally take care of updating this, but as this is moving from a manual -- automated update, it'll need to be updated manually for now

Reply:
I'm closing this PR. I updated more configuration and moved that to the PR
https://github.com/wolfi-dev/os/pull/30869
@mamccorm Can you provide your feedback ?